app/models/order_detail.rb

```diff
@@ -250,7 +250,7 @@ def can_be_viewed_by?(user)
 
   # Supports single type or an array of product types
   scope :for_product_type, ->(product_type) { joins(:product).where(products: { type: product_type }) }
-  scope :untimed_orders, -> { for_product_type(["Item", "Service"]) }
+  scope :item_and_service_orders, -> { for_product_type(["Item", "Service", "TimedService"]) }
```

Comment:
is there a categorization here that could explain a bit more about why these are grouped together? "untimed" gave a little hint as to what the purpose of the group was